ramls/loan.json

```diff
@@ -53,6 +53,10 @@
       "description": "ID of last policy used in relation to this loan",
       "type": "string"
     },
+    "loanPolicyName": {
```

Comment:
This should be marked as read only because it is derived by the server.

Please also update the comments to reflect that this is a derived property. Please refer to this [example](https://github.com/folio-org/mod-circulation/blob/666d15a77614d55ae66cde6ac07f050e3fba876f/ramls/request.json#L137)

Also this property should be ignored on incoming representations of loans.

Reply:
Could you please elaborate on which changes are needed with regard to your comment about ignoring the property on incomming representation of loans?

Reply:
Sure. 

When a client fetches a loan, this includes the derived summary properties that help it make decisions or present additional information.

Rather than require the client to remove these properties before submitting a PUT to the loan, mod-circulation should remove these properties gracefully before attempting to store the loan.

This is currently done [here](https://github.com/folio-org/mod-circulation/blob/666d15a77614d55ae66cde6ac07f050e3fba876f/src/main/java/org/folio/circulation/domain/LoanRepository.java#L215)

Please also include the {{borrower}} property as well, that I missed from the previous code review.

Does that make sense?